x-pack/legacy/plugins/lens/public/indexpattern_plugin/operations/definitions/date_histogram.tsx

```diff
@@ -197,7 +190,7 @@ export const dateHistogramOperation: OperationDefinition<DateHistogramIndexPatte
         {currentColumn.params.interval !== autoInterval && (
           <EuiFormRow
             label={i18n.translate('xpack.lens.indexPattern.dateHistogram.interval', {
-              defaultMessage: 'Time intervals',
+              defaultMessage: 'Time interval',
```

Comment:
We should use the same wording here as in the [old editor](https://github.com/elastic/kibana/blob/c1fd6eb9b4892f2b63ae4c07ca5b669003e9c415/src/legacy/ui/public/vis/editors/default/controls/time_interval.tsx#L130) here, since we don't actually guarantee that we will use that exact interval, but this interval or an interval larger than that (in case we would generate too many buckets otherwise). We've seen in the old editor that users were often confused before the wording change, why they configure a time interval and we use a different one nevertheless.